### PR TITLE
contributing.md - seek early feedback

### PR DESCRIPTION
**Pull request by jamestalmage**
As requested: https://github.com/sindresorhus/ava/pull/346#issuecomment-166789457

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ We have a [chat](https://gitter.im/sindresorhus/ava). Jump in there and lurk, ta
 ### Submitting a pull request
 
 - Non-trivial changes are often best discussed in an issue first, to prevent you from doing unnecessary work.
+- For ambitious tasks, you should try to get your work in front of the community for feedback as soon as possible. Open a PR as soon as you have done the minimum needed to demonstrate your idea. At this early stage, don't worry about making things perfect, or 100% complete. Add a [WIP] prefix to the title, and describe what you still need to do. This lets reviewers know not to nit-pick small details or point out improvements you already know you need make.
```
Comment (sindresorhus) [inline contributing.md]: `you need make` => `you need to make`

Comment (sindresorhus) [inline contributing.md]: `PR` => `pull request`

**Comment by jamestalmage** [comment]
@sindresorhus - updated.

**Comment by sindresorhus** [comment]
Excellent.

**Comment by BarryThePenguin** [comment]
For clarification, should an issue be raised for discussion first, then open a PR?

**Comment by jamestalmage** [comment]
> should an issue be raised for discussion first, then open a PR

Oh, absolutely. Does the line above not make that clear? Should we amend this:

> For ambitious tasks, you should try to get your work in front of the community ...

To 

> Once consensus has been reached in the issue thread, and you've begun working on a solution, you should try to get your work...

**Comment by jamestalmage** [comment]
Should have waited for a bit more consensus on this PR! :wink:

**Comment by BarryThePenguin** [comment]
Ah! I see it now, it's the first point. All good then..

Though the dot points are quite wordy, dot points should be short and sharp eg:

> Open an issue. Non-trivial changes should be discussed first, to prevent you from doing unnecessary work.

**Comment by sindresorhus** [comment]
> Should have waited for a bit more consensus on this PR!

Ah, yes, but it looked pretty clear cut. Will give it some more time in the future.

**Comment by jamestalmage** [comment]
I thought so to, I was just pointing out the irony that we rushed the PR that said "seek feedback ... find consensus".

**Comment by sindresorhus** [comment]
Lol, that's true :p

**Pull request merged**